### DISCUSSION TITLE
ContextMenu - What does `ForceMinimumPosToZero` do?

### DISCUSSION THREAD

**Discussion by tig** [tui-cs/Terminal.Gui#2128] [Q&A] [answered]
<img width="371" alt="image" src="https://user-images.githubusercontent.com/585482/197257475-60cded0b-1c1d-4e9d-9d3b-99dbc9c83123.png">

I have read this several times, and played with the `ContextMenu` sample, and still do not understand what it does.

@BDisp - can you explain? I'll update the docs once I understand it. Thank.s

**Comment by BDisp** [comment]
> <img alt="image" width="371" src="https://user-images.githubusercontent.com/585482/197257475-60cded0b-1c1d-4e9d-9d3b-99dbc9c83123.png">
> 
> I have read this several times, and played with the `ContextMenu` sample, and still do not understand what it does.
> 
> @BDisp - can you explain? I'll update the docs once I understand it. Thank.s

When resizing the terminal force the context menu stay at X = 0 or allow negative values like X = -2. I could be rename it to ForceLeftPositionOnResize at the time, but was the name that come to my mind.

![context-menu](https://user-images.githubusercontent.com/13117724/197263819-f1d22343-e0e5-43b3-ae79-3a021c8cd86c.gif)

**Comment by tig** [comment, marked as answer]
Got it.

To verify, is this accurate?

```		/// <summary>
		/// Sets or gets whether the context menu will be forced to the right, ensuring it is not clipped, if the x position 
		/// is less than zero. The default is <see langword="true"/> which means the context menu will be forced to the right.
		/// If set to <see langword="false"/>, the context menu will be clipped on the left if x is less than zero.
		/// </summary>
```
